vlib/v/gen/native/tests/struct.vv

```diff
@@ -196,9 +196,45 @@ fn assign_fields() {
 	assert f.a == 2
 }
 
+struct Nest {
+        a string
+}
```

Comment:
Can you add a `before string` and `after string` here too, to make sure that the offset of the inner field is correct?